### PR TITLE
Fix backup-volfile-server(s) in mount provider

### PR DESCRIPTION
**Pull request by vchung-nz**
Fix case statement and use backupvolfile_servers parameter for multiple servers.

### CODE REVIEW

File: providers/mount.rb [hunk 1]
```diff
@@ -81,11 +81,11 @@ def basic_mount_options
 end
 
 def mount_options_for_backup_server
-  case new_resource.backup_server.class
+  case new_resource.backup_server
   when String
     ',backupvolfile-server=' + new_resource.backup_server
   when Array
-    ',backupvolfile-server=' + new_resource.backup_server.join(',backupvolfile-server=')
+    ',backup-volfile-servers=' + new_resource.backup_server.join(':')
```
Comment (shortdudey123) [inline providers/mount.rb]: `backup-volfile-servers` is not correct per docs

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 - **[PR #67](https://github.com/shortdudey123/chef-gluster/pull/67)** - Fix serverspec verification
 - **[PR #68](https://github.com/shortdudey123/chef-gluster/pull/68)** - Add kitchen testing for gluster::client recipe
 - **[PR #69](https://github.com/shortdudey123/chef-gluster/pull/69)** - Update yum_repository resource
+- **[PR #70](https://github.com/shortdudey123/chef-gluster/pull/70)** - Fix backup-volfile-server(s) in mount provider
```
Comment (shortdudey123) [inline CHANGELOG.md]: This is PR 72, not 70

File: providers/mount.rb (hunk 1 above)
Comment (vchung-nz) [inline providers/mount.rb]: Could be a platform specific addition?  man mount.glusterfs (glusterfs-fuse-3.7.15-1.el7.x86_64) has the following parameter:

      backup-volfile-servers=SERVERLIST
              Provide list of backup volfile servers in the following format [default: None]

Should I remove this?

Comment (shortdudey123) [inline providers/mount.rb]: hmm interesting, i find no mention of `backup-volfile-servers` on https://gluster.readthedocs.io/en/latest/Administrator%20Guide/Setting%20Up%20Clients/

Comment (shortdudey123) [inline providers/mount.rb]: It does look like you are correct though
https://github.com/gluster/glusterfs/blob/2e23c62cc50037c8e61bcd9c04348409e7627181/xlators/mount/fuse/utils/mount_glusterfs.in#L266

Comment (shortdudey123) [inline providers/mount.rb]: 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,7 @@
 - **[PR #67](https://github.com/shortdudey123/chef-gluster/pull/67)** - Fix serverspec verification
 - **[PR #68](https://github.com/shortdudey123/chef-gluster/pull/68)** - Add kitchen testing for gluster::client recipe
 - **[PR #69](https://github.com/shortdudey123/chef-gluster/pull/69)** - Update yum_repository resource
+- **[PR #72](https://github.com/shortdudey123/chef-gluster/pull/70)** - Fix backup-volfile-server(s) in mount provider
```
Comment (shortdudey123) [inline CHANGELOG.md]: Link and PR number don't match :)

Comment (vchung-nz) [inline CHANGELOG.md]: Updated to PR #72 now.

**Comment by shortdudey123** [comment]
Sorry, one last thing...
Can you rebase on master then move the changelog entry to the `Unreleased` section?
Will merge after that 😄

**Comment by shortdudey123** [comment]
Can you fix the merge conflict?
```
Conflicting files
CHANGELOG.md
```

**Comment by vchung-nz** [comment]
File updated

**Comment by shortdudey123** [comment]
can you squash the commits so the sync with master is not in the commit history? thanks!

**Pull request merged**